### ISSUE TITLE
[Bug]: Baxter default colours cause buttons to look invisible.

### ISSUE THREAD

**Issue by sverleis** [Automattic/themes#6947]
### Quick summary

When using the Baxter theme, all store buttons are the same default colour in both text and background, making them invisible to typical users:

![image](https://user-images.githubusercontent.com/16252298/227559805-c20ddf77-2f3d-4233-a30f-2c48cf2d26b1.png)

This can easily be updated via the template, but by default, the button's background colour should be the theme's foreground colour.

![image](https://user-images.githubusercontent.com/16252298/227562820-ef4c9970-8fb3-4dc7-93ee-4ccf6eb90ec3.png)

This is for all buttons when using the theme.

### Steps to reproduce

Load up Baxter theme.

Add any button, or locate a product which will have an add-to-cart button.

### What you expected to happen

The default colours for buttons should match the default colours of the theme, at least in having one of them show the darker colour to make the button visible.

### What actually happened

Button is rendering with button text, border and background all matching the theme backgound colour.

### Browser

Google Chrome/Chromium

### Context

Customer report, and subsequent testing on my sites.

### Platform (Simple, Atomic, or both?)

Atomic

### Reproducibility

Consistent

### Severity

One

### Available workarounds?

Yes, easy to implement

### Workaround details

User to update the template colours for the theme.
